content/setup.rst

```diff
@@ -0,0 +1,96 @@
+Workshop setup
```

Comment:
Should this be included in the toctree? Or is it enough to refer to it in `index.rst`?

Reply:
last night i didn't think so, but now i do! updates to follow